### PR TITLE
[backend] chore: remove OpenAEV registration feature flag

### PR DESCRIPTION
**Pull request by Kroustille**
# Context
> Briefly describe the purpose and context of this PR. Include relevant screenshots or screen recordings for the product team.

- remove OpenAEV registration feature flag

- update or add a new user
- check that `MANAGE_OPENAEV_REGISTRATION` capability is available

## Related issues

- Related to #997 

# Checklist

## Quality

- [x] I consider this work complete and ready for review
- [x] I tested all features and edge cases
- [x] I refactored code where necessary to improve quality
- [x] I made sure my code meets development guidelines
- [x] I performed a self-review of my code

## Testing

- [ ] I wrote test cases for the relevant use cases (unit, integration or e2e tests)
- [ ] (Bug fixes only) I added tests that reproduce and verify the fix

### What tests have been made

- [ ] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [x] Local manual tests

# Additional information
> Any additional context, dependencies, or concerns reviewers should know about.

### CODE REVIEW

File: apps/portal-front/src/components/service/document/shareable-resource-slug.tsx [hunk 1]
```diff
@@ -64,14 +63,11 @@ const ShareableResourceSlug: React.FunctionComponent<
...
   };
-  const isOneOpenAEVRegistrationFeatureEnabled = useIsFeatureEnabled(
-    FeatureFlag.OPENAEV_REGISTRATION
-  );
   const shouldShowOneClickDeployComponent = useMemo(() => {
     if (!documentData.active) return false;
 
     if (documentData.type === ShareableResourceType.OPENAEV_SCENARIO) {
-      return isOneOpenAEVRegistrationFeatureEnabled;
+      return true;
```
Comment (carinelebas) [inline apps/portal-front/src/components/service/document/shareable-resource-slug.tsx:70]: It would be better to add `ShareableResourceType.OPENAEV_SCENARIO` in the array below
